data/islamabad-infobox.txt

```diff
@@ -0,0 +1,83 @@
+{{Infobox settlement
```

Comment:
Why do we need both TXT files? Will the infobox suffice for the test? If so. Let's remove the big one. We should only need the file that is used in the test.